api/external/compliance/reporting/reporting.proto

```diff
@@ -1021,7 +1022,20 @@ message Assets {
 	//Last run details of asset
 	string last_run = 4;
 }
+
 message AssetListResponse {
 	//Assets list as per the filters applied
 	repeated Assets assets = 1;
 }
+
+message ComplianceConfigRequest{
```

Comment:
Please add come comments to update the swagger documentation. @SahithiMy  @daveaugustus